datumaro/plugins/vgg_face2_format.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (C) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import csv
+import os
+import os.path as osp
+import re
+from glob import glob
+
+from datumaro.components.converter import Converter
+from datumaro.components.extractor import (AnnotationType, Bbox, DatasetItem,
+    Importer, Points, PointsCategories, SourceExtractor)
+
+
+class VggFace2Path:
+    ANNOTATION_DIR = "bb_landmark"
+    IMAGE_EXT = '.jpg'
+    BBOXES_FILE = 'loose_bb_'
+    LANDMARKS_FILE = 'loose_landmark_'
+
+class VggFace2Extractor(SourceExtractor):
+    def __init__(self, path):
+        if not osp.isfile(path):
+            raise Exception("Can't read annotation file '%s'" % path)
+        self._path = path
+        self._dataset_dir = osp.dirname(osp.dirname(path))
+
+        subset = osp.splitext(osp.basename(path))[0]
+        match = re.fullmatch(r'loose_landmark_\S+', subset)
+        if match:
+            subset = subset.split('_')[2]
+        super().__init__(subset=subset)
+
+        self._load_categories()
+        self._items = list(self._load_items(path).values())
+
+    def _load_categories(self):
+        self._categories[AnnotationType.points] =  PointsCategories.from_iterable([
+                (0, None, [[0, 1], [1, 2], [2, 3], [3, 4]])])
+
+    def _load_items(self, path):
+        items = {}
+        with open(path) as content:
+            landmarks_table = list(csv.DictReader(content))
+
+        for row in landmarks_table:
+            item_id = row['NAME_ID']
+            image_path = osp.join(self._dataset_dir, self._subset, item_id
+                + VggFace2Path.IMAGE_EXT)
+            annotations = []
+            if len([p for p in row if row[p] == '']) == 0:
```

Comment:
There is a visibility flag for missing points. Also we need to check if the number of points is different from 5.